### PR TITLE
add example for apiCheck remote env var

### PR DESCRIPTION
**Pull request by Dnhem**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [X] Docs
* [ ] Learn
* [ ] Other

## Notes for the Reviewer
Draft PR for previewing purposes

### CODE REVIEW

File: site/content/docs/cli/using-environment-variables.md [hunk 1]
```diff
@@ -74,6 +74,31 @@ test('Check Home Page', async ({ page }) => {
   await page.screenshot({ path: 'home.jpg' })
 })
 ```
+Here is an example of a Checkly [ApiCheck construct](https://www.checklyhq.com/docs/cli/constructs-reference/#apicheck) using an `ENVIRONMENT_URL` variable to define the page to visit. Notice the `{{ENVIRONMENT_URL}}` is wrapped in double handlebars and must be written this way for a remote environment variable to be parsed within an ApiCheck construct.
+
+```ts
```
Comment (tnolet) [inline site/content/docs/cli/using-environment-variables.md:79]: @Dnhem we use single quotes for strings, e.g. `'` and don't terminate the line with a `;`

**Review by tnolet — CHANGES_REQUESTED** [review]
small nit on code linting

File: site/content/docs/cli/using-environment-variables.md (hunk 1 above)
Comment (Dnhem) [reply site/content/docs/cli/using-environment-variables.md:79]: @tnolet Cleaned it up 🧹

File: site/content/docs/cli/using-environment-variables.md [hunk 2]
```diff
@@ -74,6 +74,31 @@ test('Check Home Page', async ({ page }) => {
...
+import * as path from 'path';
```
Comment (tnolet) [inline site/content/docs/cli/using-environment-variables.md:80]:
```suggestion
import * as path from 'path'
```

File: site/content/docs/cli/using-environment-variables.md [hunk 3]
```diff
@@ -74,6 +74,31 @@ test('Check Home Page', async ({ page }) => {
...
+import { ApiCheck, AssertionBuilder } from 'checkly/constructs';
+import { websiteGroup } from './website-group.check';
```
Comment (tnolet) [inline site/content/docs/cli/using-environment-variables.md:82]:
```suggestion
import { websiteGroup } from './website-group.check'
```

**Review by tnolet — CHANGES_REQUESTED** [review]
you left two `;`.

File: site/content/docs/cli/using-environment-variables.md (hunk 2 above)
Comment (Dnhem) [reply site/content/docs/cli/using-environment-variables.md:80]: @tnolet DOH! Sorry

**Review by tnolet — APPROVED** [review]
LGTM

**Pull request merged**